aspnet/mvc/models/model-binding.rst

```diff
@@ -74,6 +74,8 @@ Binding formatted data from the request body
 
 Request data can come in a variety of formats including JSON, XML and many others. When you use the [FromBody] attribute to indicate that you want to bind a parameter to data in the request body, MVC uses a configured set of formatters to handle the request data based on its content type. By default MVC includes a ``JsonInputFormatter`` class for handling JSON data, but you can add additional formatters for handling XML and other custom formats. 
 
+.. Note:: There can be at most one parameter decorated with ``[FromBody]``. The ASP.NET Core MVC run-time delegates the responsibility of reading the request stream to the formatter. Once the request stream is read for a parameter, it's generally not possible to read the request stream again for binding other ``[FromBody]`` parameters.
```

Comment:
This needs to say "at most one parameter **per action**". Other than that looks good